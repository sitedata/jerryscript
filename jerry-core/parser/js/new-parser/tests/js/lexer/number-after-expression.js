/* This test contains number literal parser tests. */
 23.5e+5	x
