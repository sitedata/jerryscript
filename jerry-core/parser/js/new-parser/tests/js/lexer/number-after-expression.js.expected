
--- Script parsing start ---

  [  1] CBC_PUSH_LITERAL idx:0->number(23.5e+5)
  [  0] CBC_POP_BLOCK

--- Script parsing end ---

Parse error 'Expected ';' token.' at line: 2 col: 17
